libverify/verify_fuzzer.h

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
@huxinwen1994 Can we move all these declarations to the .cpp file?

Reply:
OK